runtimes/protocol/lsp.ts

```diff
@@ -107,5 +108,14 @@ export interface InitializeResult extends _InitializeResultBase {
      */
     awsServerCapabilities?: {
         chatOptions?: ChatOptions
+        configurationProvider?: ConfigurationOptions
     }
 }
+
+/**
+ * Configuration options for AWS Runtimes
+ * Sent back to the client to signal available configuration values
+ */
+export interface ConfigurationOptions {
+    configurations: string[]
```

Comment:
Is this suppose to be a list of supported `sections`, to which server can reply with configurations? Then i'd suggest to change `configurations` to `sections` to be more explicit